java_training/src/com/lftechnology/december31/DateTest.java

```diff
@@ -0,0 +1,133 @@
+package com.lftechnology.december31;
+
+/**
+ * Write a test application named DateTest that demonstrates class Date's
+ * capabilities.
+ * 
+ * @author nerisa
+ * 
+ */
+public class DateTest {
+
+	public static void main(String[] args) {
+		DateUtility date = new DateUtility(2, 1, 2015);
```

Comment:
what if i entered 13 in the month parameter, it would display 0 month i guess please check it